### ISSUE TITLE
[FEAT]: Folder structure on NextCloud

### ISSUE THREAD

**Issue by gcielniak** [KarolineH/projectpage#4]
### Description

I propose a simple folder structure for the dataset to separate files of individual plants. Something simple like:
 - LAST-Straw/AX + BX
 - LAST-Straw/Katrina/AX, LAST-Straw/Zara/BX.

Good idea? And if yes, which option?

**Comment by KarolineH** [comment]
Agree, but have no strong preference on either option. @KatherineJames?
